### PR TITLE
Ddcnl 3108

### PR DESCRIPTION
**Pull request by thirlz95**

### CODE REVIEW

**Review by matthew-mcnally — APPROVED** [review]

File: app/forms/coc/DivorceSelectYearForm.scala [hunk 1]
```diff
@@ -30,7 +30,7 @@ import scala.util.Try
 
 object DivorceSelectYearForm {
 
-  val DateOfDivorce = "dateOfDivorce"
+  val  DateOfDivorce = "dateOfDivorce"
```
Comment (gianick) [inline app/forms/coc/DivorceSelectYearForm.scala:33]:
```suggestion
  val DateOfDivorce = "dateOfDivorce"
```

Comment (Tateni93) [inline app/forms/coc/DivorceSelectYearForm.scala:33]: Double space here

File: app/views/coc/divorce_select_year.scala.html [hunk 2]
```diff
@@ -49,8 +49,10 @@ <h1 class="heading-xlarge">@messages("pages.divorce.title")</h1>
             DivorceSelectYearForm.DateOfDivorce,
             Html(""),
             None,
+            '_legend -> Messages("pages.divorce.title"),
```
Comment (Tateni93) [inline app/views/coc/divorce_select_year.scala.html:52]: can we use the implicit messages here
```suggestion
            '_legend -> messages("pages.divorce.title"),
```

File: app/views/coc/divorce_select_year.scala.html [hunk 3]
```diff
@@ -49,8 +49,10 @@ <h1 class="heading-xlarge">@messages("pages.divorce.title")</h1>
...
+            '_legendClass -> "visuallyhidden",
             '_hintText -> Messages("pages.divorce.date.hint"),
-            '_label -> Messages("collect_user_data.dod.error"),
+            '_label -> Html(Messages("collect_user_data.dod.error")),
```
Comment (Tateni93) [inline app/views/coc/divorce_select_year.scala.html:55]:
```suggestion
            '_label -> Html(messages("collect_user_data.dod.error")),
```

**Review by gianick — APPROVED** [review]

File: app/views/multiyear/do_you_live_in_scotland.scala.html [hunk 4]
```diff
@@ -43,7 +43,7 @@ <h1 class="heading-xlarge">@Html(Messages("eligibility.check.do.you.live.in.scot
         @templates.input_radio_group(
         doYouLiveInScotlandForm("do-you-live-in-scotland"),
         Seq(("true", Messages("radio.yes")), ("false", Messages("radio.no"))),
-        '_legend -> Html(Messages("eligibility.check.do.you.live.in.scotland.legend")),
+        '_legend -> Html(Messages("eligibility.check.do.you.live.in.scotland.h1")),
```
Comment (Tateni93) [inline app/views/multiyear/do_you_live_in_scotland.scala.html:46]: Use implicit messages
```suggestion
        '_legend -> Html(messages("eligibility.check.do.you.live.in.scotland.h1")),
```

**Review by Tateni93 — APPROVED** [review]

**Pull request merged**